CHANGELOG.md

```diff
@@ -179,6 +189,7 @@ chart-operator).
 
 
 
+[v3.3.1]: https://github.com/giantswarm/k8scloudconfig/commits/master/v_3_3_1
```

Comment:
`v3.3.2` is missing, which you just added.

Reply:
Good catch. Done.